ush/setup.py

```diff
@@ -1122,14 +1163,24 @@ def _get_location(xcs, fmt, expt_cfg):
     do_ensemble = global_sect["DO_ENSEMBLE"]
     if (not do_ensemble) and ens_vx_tasks:
         task_str = "    " + "\n    ".join(ens_vx_tasks)
-        msg = dedent(f"""
+        msg = dedent(
+            f"""
               Ensemble verification can not be run unless running in ensemble mode:
                   DO_ENSEMBLE = \"{do_ensemble}\"
               Ensemble verification tasks:
-              """)
-        msg = "".join([msg, task_str, dedent(f"""
+              """
+        )
+        msg = "".join(
+            [
+                msg,
+                task_str,
+                dedent(
+                    f"""
               Please set DO_ENSEMBLE to True or remove ensemble vx tasks from the
-              workflow.""")])
+              workflow."""
+                ),
+            ]
+        )
         raise Exception(msg)
```

Comment:
I'm not sure this captures the full intent, but could this just be
```
msg = f'''
Ensemble verification cannot be run unless running in ensemble mode:
    DO_ENSEMBLE = "{do_ensemble}"
Ensemble verification tasks:
    {task_str}
Please set DO_ENSEMBLE to True or remove ensemble vx tasks from the workflow.
'''
raise Exception(dedent(msg).strip())
```
?

Reply:
I didn't even notice this one. Absolutely.